projects/libressl/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER ab@pompel.me
+RUN apt-get update && apt-get install -y make cmake
+RUN git clone --depth 1 https://github.com/libressl-portable/portable.git libressl
+RUN git clone --depth 1 https://github.com/doyensec/libressl-portable.git libressl.fuzzers
```

Comment:
Can these fuzzers be upstreamed?

Reply:
Sure, I'm working on upstreaming them:

https://marc.info/?l=libressl&m=154694115125905&w=2

Once upstreamed the `21` line in `projects/libressl/Dockerfile` should be deleted and the `projects/libressl/build.sh` should be changed with something like `sed -i "s|libressl.fuzzers|libressl|g" projects/libressl/build.sh`